src/figure_kind_randomizer.py

```diff
@@ -0,0 +1,15 @@
+from figure_kind import FigureKind
```

Comment:
please review this file since it is out of scope

Reply:
This file is ok, I want to merge it and assign to Serg

Reply:
> This file is ok, I want to merge it and assign to Serg

do it in another PR please